Assets/_Project/Scripts/Level/LevelBrowserManager.cs

```diff
@@ -86,6 +89,23 @@ private void OnLevelDownloadRequested(string code)
         AnalyticsManager.Instance.TrackEvent(AnalyticsEvent.LevelDownload, code);
     }
 
+    private void OnLevelSearchRequested(string code)
+    {
+        var remoteLevelLoadQuery = this.LevelQuery;
+        remoteLevelLoadQuery.code = code;
+        this.LevelQuery = remoteLevelLoadQuery;
+        DestroyAndRefreshLevels();
+    }
+
+    private void OnLevelSortRequested(string sort, SortDirection direction = SortDirection.Ascending)
+    {
```

Comment:
@CalmPewter This method is not hooked up yet. Changing the sort of the level browser should end up triggering this code.